signing-manifests/bug1835022.yml

```diff
@@ -0,0 +1,13 @@
+---
+bug: 1835022
+sha256: f2bd4983f08135a6cccb85bc45a060a76584598620698ad2ffd48e836d9e94c0
+filesize: 1422
+private-artifact: false
+signing-formats: ["autograph_hash_only_mar384"]
+signing-cert: release-signing
+requestor: Geoff Brown <gbrown@mozilla.com>
+reason: channel-switching from mozilla-release to mozilla-esr115
+artifact-name: switch-to-esr115.mar
+fetch:
+    type: static-url
+    url: https://github.com/mozilla-releng/adhoc-signing-blobs/raw/mar/switch-to-esr115.mar
```

Comment:
Could use a bmo-attachment instead of static-url since the file is attached to the bug.

Reply:
Thanks. I forgot about the bmo-attachment type...thought it was strange I couldn't use an attachment!